src/Bundle/Maker/MakeFactory.php

```diff
@@ -80,18 +90,19 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
     {
         $command
             ->setDescription(self::getCommandDescription())
-            ->addArgument('entity', InputArgument::OPTIONAL, 'Entity class to create a factory for')
+            ->addArgument('class', InputArgument::OPTIONAL, 'Entity, Document or class to create a factory for')
             ->addOption('namespace', null, InputOption::VALUE_REQUIRED, 'Customize the namespace for generated factories', 'Factory')
             ->addOption('test', null, InputOption::VALUE_NONE, 'Create in <fg=yellow>tests/</> instead of <fg=yellow>src/</>')
             ->addOption('all-fields', null, InputOption::VALUE_NONE, 'Create defaults for all entity fields, not only required fields')
+            ->addOption('not-persisted', null, InputOption::VALUE_NONE, 'Create a factory for an object not managed by Doctrine')
```

Comment:
not sure of the option's name :shrug:

Reply:
> not sure of the option's name

Me either, lets go with this for now.

> I'm wondering if we should not auto activated this option if doctrine is not present, WDYT?

Yep, seems reasonable.